lib/puppet/parser/functions/mco_array_to_string.rb

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env ruby
 # vim: set sw=2 sts=2 et tw=80 :
-Puppet::Parser::Functions.newfunction(:mco_array_to_string, type: :rvalue) do |args|
+Puppet::Parser::Functions.newfunction(:mco_array_to_string, :type => :rvalue) do |args|
```

Comment:
We do not support 1.8.7 any longer, and this will be revered with our next modulesync.